src/VCEL.Core/Lang/VCELLexer.g4

```diff
@@ -47,8 +47,8 @@ LONG_LITERAL: (DIGIT)+ LONG_SUFFIX;
 INT_LITERAL: (DIGIT)+;
 FLOAT_LITERAL: NUMBER ('f'|'F');
 DATE_LITERAL: '@' TDG TDG '-' TDG '-' TDG ('T' TIME_LITERAL (TZ)?)?;
-TZ: 'Z' | (('+'|'-') TDG (':'?) TDG?);
-TIME_SPECIFIER: 'T';
+fragment TZ: 'Z' | (('+'|'-') TDG (':'?) TDG?);
```

Comment:
Added a fragment here as otherwise the string 'Z' was always being identified as a TZ, as opposed to an identifier.